chucks-pick3-web/angular-app/src/app/components/customer-summary.component/customer-summary.component.html

```diff
@@ -1,3 +1,70 @@
-<p>
-  customer-summary works!
-</p>
+<div class="customer-title-section">
+  <h1><b>Customer Summary</b></h1>
+</div>
+<div class="form-row options">
+  <div class="form-col campaign">
+    <label for="emailCampaign">Email Campaign</label>
+    <label for="emailCampaign"></label>
+    <select #dropDownEmailCampaign="ngModel"
+            name="dropDownHowOftenName"
+            class="custom-select" id="emailCampaign"
+            required
+            ngModel>
+      <option value="" selected>No option selected</option>
+      <option value="Often">Option 1</option>
+      <option value="Sometimes">Option 2</option>
+      <option value="Never">Option 3</option>
+    </select>
+  </div>
+  <div class="buttons">
+    <div class="col-sm">
+      <button type="button"
+              class="btn btn add btn-success btn-sm btn-block" (click)="addRow(customers)">
```

Comment:
Do not pass customers to `addRow() method`.  We should be dynamically added a new customer when Add button is clicked via `addRow() method`.

Reply:
Removed customers from parentheses and made into a const.